obj_files.mk

```diff
@@ -197,7 +197,6 @@ TEXT_O_FILES +=\
     $(BUILD_DIR)/asm/melee/ft/ftstarrodswing.s.o\
     $(BUILD_DIR)/asm/melee/ft/ftlipstickswing.s.o\
     $(BUILD_DIR)/asm/melee/ft/ft_0CDD.s.o\
-    $(BUILD_DIR)/asm/melee/ft/ftattacks4combo.s.o\
```

Comment:
Change this to `$(BUILD_DIR)/asm/melee/ft/ftattacks4combo.c.o\`

Reply:
You mean `src` not `asm`: 
```
$(BUILD_DIR)/src/melee/ft/ftattacks4combo.c.o\
```